script/Ciel/Day 13/13_01_02_CIEL13_5_2.txt

```diff
@@ -526,12 +592,17 @@ I embrace Senpai's motionless body.
 [sha:9fff8e782f46c3cdc5b1c750965ca0665ae828f5]{
 -- Page 101, Offset 35549.
 -- 藍色の髪を撫でながら名前を呼びかけて、今の自分にある精一杯の言葉を伝える。
-Calling her name as I brush her blue hair, I try my best to convey all of my emotions to her.
+// Hold the fuck up, we're just going to casually acknowledge it's blue?
+// You tried to claim she was brunette not three files ago you damn hack
+Calling her name as I brush her blue hair,
+ I try as best I can to put everything I'm feeling right now into words.
```

Comment:
Clearly, we can only deduce she became blue haired post-Roa.

Also, brush > stroke